Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -705,6 +705,28 @@ UpdatePayloadId (
   }
 }
 
+//Initialize Platform Igd OpRegion
+VOID
+EFIAPI
+IgdOpRegionPlatformInit (
+  VOID
+  )
+{
+  GLOBAL_NVS_AREA           *Gnvs;
+  IGD_OP_PLATFORM_INFO      IgdPlatformInfo;
+  EFI_STATUS                Status;
+
+  Gnvs = (GLOBAL_NVS_AREA *)(UINTN)PcdGet32(PcdAcpiGnvsAddress);
+  IgdPlatformInfo.IgdOpRegionAddressPtr = (UINTN *)&(Gnvs->SaNvs.IgdOpRegionAddress);
+  IgdPlatformInfo.TurboIMON = Gnvs->SaNvs.GfxTurboIMON;
+  IgdPlatformInfo.callback = NULL;
+
+  Status = IgdOpRegionInit(&IgdPlatformInfo);
+  if (EFI_ERROR (Status)) {
+    DEBUG ((DEBUG_WARN, "VBT not found %r\n", Status));
+  }
+}

```

Comment:
"What about adding a new GetIgdOpRegionAddress() API?"

> What about adding a new GetIgdOpRegionAddress() API?
> 
> ```
> Status = IgdOpRegionInit(&IgdPlatformInfo);
> if (!EFI_ERROR (Status)) {
>   Status = GetIgdOpRegionAddress (&Address);
>   if (!EFI_ERROR (Status)) {
>     Gnvs->SaNvs.IgdOpRegionAddress = Address;
>   }
> }
> ```
> 
> If TurboIMON TRUE and callback NULL is default on most of platforms, `IgdOpRegionInit (NULL)` would be simple.
> 
> And, please follow EDKII coding convention - a space between API name and ().

Can we use a PCD for the IGD op region address instead of a new API function ?   It will be simpler to handle.

Reply:
This is great idea! Why I never thought of it. Thanks

Reply:
@mauricema done

Reply:
> What about adding a new GetIgdOpRegionAddress() API?
> 
> ```
> Status = IgdOpRegionInit(&IgdPlatformInfo);
> if (!EFI_ERROR (Status)) {
>   Status = GetIgdOpRegionAddress (&Address);
>   if (!EFI_ERROR (Status)) {
>     Gnvs->SaNvs.IgdOpRegionAddress = Address;
>   }
> }
> ```
> 
> If TurboIMON TRUE and callback NULL is default on most of platforms, `IgdOpRegionInit (NULL)` would be simple.
> 
> And, please follow EDKII coding convention - a space between API name and ().

Good point to save redundant codes in platform codes. Removed. Will take note of similar approach next time. Thanks!